resources/objects/furniture/BunBottomBox.tscn

```diff
@@ -57,11 +57,14 @@ item_scene = ExtResource("2_3uq4g")
 
 [node name="BunBottom" parent="Content" instance=ExtResource("2_3uq4g")]
 transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0.0345346, 0)
+ranged_grab_method = 1
 
 [node name="BunBottom2" parent="Content" instance=ExtResource("2_3uq4g")]
 transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0.0690693, 0)
+ranged_grab_method = 1
 
 [node name="BunBottom3" parent="Content" instance=ExtResource("2_3uq4g")]
 transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0.101867, 0)
+ranged_grab_method = 1
```

Comment:
why ranged grab as snap here? Or Why range grab at all?

Reply:
This was a left-over from changing it to lerp once. It is deactivated in the hands anyway